src/MemoryTracing/MemoryTracingUtilsTest.cpp

```diff
@@ -28,13 +28,13 @@ MATCHER_P(MemoryProtoEq, expected, "") {
 namespace orbit_memory_tracing {
 
 TEST(MemoryUtils, UpdateSystemMemoryUsageFromMemInfo) {
-  const int kMemTotal = 16396576;
-  const int kMemFree = 11493816;
-  const int kMemAvailable = 14378752;
-  const int kBuffers = 71540;
-  const int kCached = 3042860;
+  const int mem_total = 16396576;
+  const int mem_free = 11493816;
+  const int mem_available = 14378752;
+  const int buffers = 71540;
+  const int cached = 3042860;
```

Comment:
I think you could make these constexpr instead and avoid the renaming.

Reply:
Done.